driver/dostuff.py

```diff
@@ -40,11 +43,19 @@
                ]
 Engines, NumUpdated = builders.build(KnownEngines, not options.noupdate, options.force)
 
+Submitter = None
+if options.submitter == 'remote':
+    Submitter = submitter.RemoteSubmitter
+elif options.submitter == 'print':
+    Submitter = submitter.PrintSubmitter
+else:
+    raise Exception('unknown submitter!')
+
 # No updates. Report to server and wait 60 seconds, before moving on
 if NumUpdated == 0 and not options.force:
     for slave in slaves.init():
-        submit = submitter.Submitter(slave)
-        submit.Awake();
+        remotecontroller = RemoteController(slave)
+        remotecontroller.Awake();
```

Comment:
Can you use Submitter for this too?
And test if submitter is 'remote' before sending Awake.